python/sglang/srt/managers/scheduler_metrics_mixin.py

```diff
@@ -184,8 +184,9 @@ def log_decode_stats(
             msg += f"pre-allocated usage: {self.disagg_decode_prealloc_queue.num_tokens_pre_allocated / self.max_total_num_tokens:.2f}, "
             msg += f"#retracted-req: {len(self.disagg_decode_prealloc_queue.retracted_queue)}, "
 
+        graph_msg = f"cpu torch-compile" if self.device == "cpu" else "cuda graph"
         msg += (
-            f"cuda graph: {can_run_cuda_graph}, "
+            f"{graph_msg}: {can_run_graph}, "
```

Comment:
```suggestion
 f"{'cpu torch-compile' if self.device == 'cpu' else 'cuda graph'}: {can_run_graph}, "
```
or we just print `sefl.device` + graph: use `cpu graph` or `cuda graph` for the naming, skip torch compile.

Reply:
Use `f"{'cpu graph' if self.device == 'cpu' else 'cuda graph'}: {can_run_graph}, "` instead.

Reply:
Use `cpu graph of torch.compile` to distinguish it from `cuda graph`.